bpf/tools/log2scapy.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+import argparse
+from scapy.all import *
+
+def parse_pkts(filename: str) -> List[Dict]:
+    #TODO attempt to parse FILENAME:LINENUM if possible (HEXDUMP())
+    #TODO parse timestamp
+    pattern = re.compile(r"\s*(.*)\s*pkt_hex\s*(\w+)\[(.*?)\]")
```

Comment:
nit: might want to put the format and an example in the doc string (regex tends to get somewhat hard to decipher over time).

Reply:
Added [here](https://github.com/cilium/cilium/pull/40294/commits/8ef304b48d499a350e7d8bffcc8907af333874f0#diff-68716399c65f043968da9c5339c8520fb00c7e883ee7c3fd2836e15bcdd1e58bR12-R19) and [here](https://github.com/cilium/cilium/pull/40294/commits/430bb1dea75066172f0b73ca7c9d983aa559c280#diff-a2e7f22890560c0d9bfcc7983e890dfafcd5c22b9d08a232ba363ddd3a537848R13-R23).